tests/models/test_connectionconfig.py

```diff
@@ -84,6 +84,6 @@ def test_create_connection_config_errors(self, db: Session, connection_config):
             )
         assert (
             str(exc.value)
-            == "Key my_postgres_db_1 already exists in ConnectionConfig. Keys will be snake-cased names if not provided. "
+            == "Key my_postgres_db_1 already exists in ConnectionConfig. Keys will be snake-cased keys if not provided. "
```

Comment:
I don't think this global find and replace from names -> keys is a good idea - this edit is not accurate - if a user is creating a connection config, and they don't supply a "key" in the request_body, we do create a key from the snake_cased name.

I've made this comment elsewhere, I'd like to not use "keys" as a variable name for these nested fields, we're overloading the term.